### PR TITLE
Change status to ReadyToUse to true for VolumeSnapshot

### PR DESCRIPTION
**Pull request by bathina2**
## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [x] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

## Issues

- #XXX

## Test Plan

- [x] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: pkg/kube/snapshot/snapshot_alpha.go [hunk 1]
```diff
@@ -208,6 +208,21 @@ func (sna *SnapshotAlpha) CreateFromSource(ctx context.Context, source *Source,
...
+		if err != nil {
+			return err
+		}
+		status, ok := us.Object["status"].(map[string]interface{})
+		if !ok {
+			errors.Errorf("Failed to convert status to map, Volumesnapshot: %s, Status: %v", snap.GetName(), status)
+		}
+		status["readyToUse"] = true
+		us.Object["status"] = status
+		if _, err := sna.dynCli.Resource(v1alpha1.VolSnapGVR).Namespace(namespace).UpdateStatus(ctx, us, metav1.UpdateOptions{}); err != nil {
```
Comment (vkamra) [inline pkg/kube/snapshot/snapshot_alpha.go:222]: Can we use `Patch` here?

Comment (bathina2) [reply pkg/kube/snapshot/snapshot_alpha.go:222]: I dont think you can patch a status. That why UpdateStatus does a PUT internally to replace the whole object

Comment (vkamra) [reply pkg/kube/snapshot/snapshot_alpha.go:222]: Got it - sounds good.

File: pkg/kube/snapshot/snapshot_alpha.go [hunk 2]
```diff
@@ -208,6 +208,21 @@ func (sna *SnapshotAlpha) CreateFromSource(ctx context.Context, source *Source,
 	if !waitForReady {
 		return nil
 	}
+	if source.Driver == "csi-vxflexos.dellemc.com" {
```
Comment (vkamra) [inline pkg/kube/snapshot/snapshot_alpha.go:211]: Minor - let's make this a constant.

Comment (bathina2) [reply pkg/kube/snapshot/snapshot_alpha.go:211]: yep will do

Comment (vkamra) [reply pkg/kube/snapshot/snapshot_alpha.go:211]: Add a comment with the link to the upstream PR here please - indicating this is a workaround till the upstream issue is addressed

Comment (vkamra) [reply pkg/kube/snapshot/snapshot_alpha.go:211]: Add a unit test please?

**Review by vkamra — APPROVED** [review]
LGTM - Unit test needed.

File: pkg/kube/snapshot/snapshot_alpha.go (hunk 2 above)
Comment (bathina2) [reply pkg/kube/snapshot/snapshot_alpha.go]: @vkamra UT'd

**Review by vkamra — APPROVED** [review]

**Pull request merged**